_guidelines-v4/05-mensural.md

```diff
@@ -10,8 +10,8 @@ sectionid: "mensural"
+There are a certain number of differences, however, regarding the representation of duration in mensural notation. The mensural module introduces specific attribute values for notes and rests in order to appropriately encode mensural durations. One of the main features is that the actual duration of a note is specified not only by its symbol but also by its position and the context in which the symbol appears. The general context is given by one of the 16 mensural *species* provided by the four levels of division: *modus major*, *modus minor*, *tempus*, and *prolatio*. Depending on the context, certain rules must be applied to determine the duration of a note. Therefore, encoding information on both the sign and its actual duration is highly desirable.
 
-Another particularity of mensural notation is the use of proportions that are indicated by numeric proportions or by specific mensuration signs. The proportions indicate that the durations have to be modified accordingly and they can be combined in a very complex manner. Over time, proportions and mensuration signs were simplified and became time signatures in CMN. The attributes and elements that are necessary for encoding proportions and mensural signs are made available by the module.
+Another feature of mensural notation is the use of proportions that are indicated by numeric proportions or by specific mensuration signs. Proportions indicate that the durations have to be modified accordingly. Proportions can be combined in a very complex manner. Over time, proportions and mensuration signs were simplified and became time signatures in CMN. The attributes and elements that are necessary for encoding proportions and mensural signs are made available by the module.
 
-In mensural notation, notes can also be notated in ligatures that regroup two or more notes. Ligatures were a legacy from an earlier notation system that were still widely used in Renaissance music notation. They gradually disappeared during the seventeenth century. The mensural module provides multiple ways of encoding the ligatures.
+In mensural notation, notes can also be notated in ligatures that regroup two or more notes. Ligatures were a legacy from earlier notation systems and they continued being widely used in Renaissance music notation. They gradually disappeared during the seventeenth century. The mensural module provides multiple ways of encoding the ligatures.
```

Comment:
regroup -> group

I don't think it's appropriate to describe them as a 'legacy.' They are codified differently in this notation than in neume notation, so they had novel use in mensural notation.

Reply:
> regroup -> group

Done! :)

> I don't think it's appropriate to describe them as a 'legacy.' They are codified differently in this notation than in neume notation, so they had novel use in mensural notation.

They are legacy from **modal notation**. The novel use was given to them in modal notation, which was inherited by the mensural system. 

I have "Ligatures were a legacy from earlier notation system**s**" (in plural) to do reference to both _modal notation_—from which their rhythmic interpretation was directly inherited—and _neumatic notation_—where they were used before, with no rhythmic interpretation of course. 

Actually, the innovation of mensural notation—the introduction of the use of different note shapes to indicate different note values—allowed to break ligatures into individual notes when needed (i.e., when each note had a different syllable assigned to it, which became a common scenario with the emergence of the motet).

Reply:
OK; Then to be clearer I think it should be explicit and not say they are from legacy notations (since that is not very specific), but from modal notation. "Ligatures, first introduced in modal notation systems as a means of notating rhythm (? revise), were adopted for use in mensural notations." 

I am not sure if "Renaissance" music notation is a well-known way of referencing this notation specifically? Are mensural notation systems used only in the Renaissance? If ligatures gradually disappeared in the 17th C. then that suggests there is some overlap when they were used with newer systems of notation / in later periods?

You do not need "the" between encoding and ligatures; it can either be "encoding these ligatures" or just "encoding ligatures."

Reply:
True, I will change this into:

> In mensural notation, notes can also be notated in ligatures that group two or more notes. Ligatures, first introduced in modal notation systems as a means of notating rhythm, were adopted for use in mensural notation and they gradually disappeared during the seventeenth century. The mensural module provides multiple ways of encoding ligatures.

To be honest, I don't have the answers to all these questions. @giulianodibacco and @karen27, ideas? (Please feel free to intervene on the other comments above.)

Reply:
I would say that "Renaissance notation" is equivalent to "white mensural notation" and this gradually transformed (decayed) into CMN during the 17th century.  White mensural notation is roughly the 15th and 16th centuries while the Renaissance is more or less the same, depending on the historian.   CMN is similar to imperfect mensurations, and compound meters such as 6/8 and 9/8 are what developed to fit perfect mensurations into the new system (in otherwise the durations of notes are contextual in mensural notation, while they are always binary in CMN).

For about 200 years preceeding white mensural notation is black mensural notation.  They are nearly equivalent and it may be related to changing writing material from animal skins to paper and the avoidance of inkblots (mentioned in this eBook](https://books.google.com/books?id=kXyFAwAAQBAJ&pg=PT1989&lpg=PT1989&dq=paper+white+mensural+notation+velum&source=bl&ots=QWsbuzMdmb&sig=ACfU3U2-K5yCud_9BTF0pUNfTmukCH8G4g&hl=en&sa=X&ved=2ahUKEwit3rXEjP3oAhVSJzQIHUpjAP0Q6AEwAHoECAgQAQ#v=onepage&q=paper%20white%20mensural%20notation%20velum&f=false)).  And prior to black mensural notation is modal notation which is much different from black/white mensural notation.

Two reasons for the decline of ligatures would be (1) increased use of barlines, since most ligatures cross barlines. and (2) scores become more common than parts for distribution of music.

Reply:
Which question is still in need of an answer? 

As for the reference to "Renaissance," I think it was in the old version of the guidelines that somehow focused on that period. I agree that it can be omitted since the sentence already concludes "gradually disappeared in the 17th-century" giving an approximate timeline, which I think this is what Martha wanted.

Reply:
@craigsapp I still find it unnecessary to give details in a simple sentence meant to give an approximate timeline from/to for the usage of ligatures

Reply:
Hi @giulianodibacco, it was the last question (about the 17th century and the decline of ligatures). Thank you! Now it is correct. 

And thanks for the clarification, @craigsapp, I didn't know about the reasons for the decline of ligatures at that time period.

I think that with the corrections pointed out by @ahankinson, the text is looking fine now. Thanks!

Reply:
In music from the [Tasso in Music Project](http://www.tassomusic.org) (1570-1640) almost all ligatures are two-note semibreve ligatures, so by that time, most ligature types were probably disappearing.  The ligatures still need to be marked up in most of the Tasso scores, so I can't do a statistical analysis of their frequency and type over that time period yet...

Looking at this book published in 1600, I see no ligatures: https://books.google.com/books?id=8MWHOfAAVYAC&printsec=frontcover&source=gbs_ge_summary_r&cad=0#v=onepage&q&f=false

If it is interesting, you can go to the browse page:   https://www.tassomusic.org/browse
and then search by a specific year to sample other dates.  Many of the work pages have links to scans of the original sources when available online, such as for the 1600 sample: https://www.tassomusic.org/work/?id=Trm0288j (The "view digitized source" link on that page).

Here is a sample from 1640: http://www.bibliotecamusica.it/cmbm/viewschedatwbca.asp?path=/cmbm/images/ripro/gaspari/_Z/Z024/
from the workpage: https://www.tassomusic.org/work/?id=Trm0047w
This one has even shorter durations for notes, about in the modern convention of using quarter notes as the beat.  Still no barlines, so probably barlines are not an influence on the decrease in ligatures, and still in parts.  Mostly it might be that the durations of the notes get so short that longer note durations becomre rare, and by extension there are less notes able to form ligatures, since the shortest notes in ligatures are semibreves.

Here is the oldest score that has an online source available: https://stimmbuecher.digitale-sammlungen.de//view?id=bsb00073081 (1575).  That interface is very bad, but looking through the tenor part I am not seeing any ligatures.  Tasso page:
https://www.tassomusic.org/work/?id=Trm0611a

Reply:
Thanks for sharing all these resources, @craigsapp! 

>  Mostly it might be that the durations of the notes get so short that longer note durations become rare, and by extension there are less notes able to form ligatures, since the shortest notes in ligatures are semibreves.

This is an interesting hypothesis. Will look into the shared resources.